contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -81,6 +81,43 @@ def _execute(self, capture_file_path: str, expect_fail=False):
             ok_button.click_input()
 
 
+class LoadLatestCapture(E2ETestCase):
+    """
+    Opens the most recent capture that matches the first given `filter_strings`. The filter strings get processed in
+    order, and the next filter string is used if no capture was found for a particular string. Also it waits until the
+    capture is loaded.
+
+    It raises an exception if no capture was found at all.
+    """
+
+    def _execute(self, filter_strings: str or List[str] or Iterable[str]):
```

Comment:
`List[str]` is unneeded here, it's a specialization of `Iterable[str]`

Reply:
Done.